packages/web/src/scss/components/CheckboxField/_CheckboxField.scss

```diff
@@ -93,11 +99,14 @@
     @include form-fields-tools.item-label-checked();
 }
 
-.CheckboxField--item .CheckboxField__message {
+.CheckboxField--item .CheckboxField__message,
+.CheckboxField--item .CheckboxField__helperText {
     @include form-fields-tools.item-message();
```

Comment:
and this mixing too. Rename to cover both texts.

Reply:
fixed

Reply:
fixed